README.md

```diff
@@ -1 +1,18 @@
 # Task-Tracker-App
+
+The app is designed to work with the [Task-Tracker-Device](https://github.com/Task-Tracker-Systems/Task-Tracker-Device).
+
+## Build
+
+### Local
+Run the gradle build in each directory or use the `build.sh` file.
+
+### Docker
+Powershell
+```Powershell
+docker build -t task-tracker-systems/task-tracker-app:1 .
```

Comment:
FWI: This took about 7mins the first time.

Reply:
That's quite slow but a one time action to build the container. Depends also a lot on your internet connection